components/Image/ui-world-map.tsx

```diff
@@ -32,8 +35,8 @@ export default function UIWorldMap({ section }: { section: uiTypes.UISectionObj
     isFirstPress: true,
     isDragging: false,
     isScaling: false,
-    divHeight: 500,
-    divWidth: 500,
+    divHeight: 350,
+    divWidth: 350,
```

Comment:
Let's get these into a const.ts or some file because I noticed the 350 is in the file above.

Reply:
Agreed, they were test values